android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -995,5 +995,13 @@ public FileStore getFileStore(StoreType storeType) {
     protected ICacheManager getSyncCacheManager() {
         return syncCacheManager;
     }
+
```

Comment:
@yuliya-guseva Please document this method, in terms of why a developer would call it.